src/table/components/TableTotals.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import TableRow from '@mui/material/TableRow';
+import TableCell from '@mui/material/TableCell';
+import { getHeaderStyle, getTotalsCellStyle } from '../utils/styling-utils';
+import { headHandleKeyPress } from '../utils/handle-key-press';
+import { handleClickToFocusHead } from '../utils/handle-accessibility';
+
+function TableTotals({
+  rootElement,
+  tableData,
+  theme,
+  layout,
+  changeSortOrder,
+  constraints,
+  translator,
+  setFocusedCellCoord,
+  keyboard,
+}) {
+  const { columns, paginationNeeded } = tableData;
+  const headerStyle = useMemo(() => getHeaderStyle(layout, theme), [layout, theme.name()]);
+  const headRowStyle = {
+    '& :last-child': {
+      borderRight: paginationNeeded && 0,
+    },
+    'th:first-of-type': {
+      borderLeft: !paginationNeeded && '1px solid rgb(217, 217, 217)',
+    },
+  };
+
+  return (
+    <TableRow sx={headRowStyle} className="sn-table-row">
+      {columns.map((column, columnIndex) => {
+        return (
+          <TableCell
+            sx={getTotalsCellStyle(headerStyle, layout)}
+            key={column.id}
+            align={column.align}
+            tabIndex={-1}
+            onKeyDown={(e) =>
+              headHandleKeyPress(
+                e,
+                rootElement,
+                [0, columnIndex],
+                column,
+                changeSortOrder,
+                layout,
+                !constraints.active,
+                setFocusedCellCoord
+              )
+            }
+            onMouseDown={() => handleClickToFocusHead(columnIndex, rootElement, setFocusedCellCoord, keyboard)}
```

Comment:
clicking the head will sort the table. Clicking the totals should just focus it

Reply:
not true, sorry. This will only handle focus, but the row index will be wrong. The row index for focus in general will be a bit tricky since it depends on how many header rows there are, that needs to be taken into account

Reply:
This one sets the rowIdx to 0, that would be wrong. So need to pass the actual cell to `clickToFocusBody`. Should proably be renamed `clickToFocusCell` if it is used here

Reply:
when you click, it sets the focus to the cell you clicked. it takes a coordinate, blurs the previous focused cell, and focus the one at given coordinate + updates the focus state. What you need to do is to figure out the correct coordinate as you do for `totalHandleKeyPress`

Tabbing has a role in the table. If you have focus on a header or body cell, tab will take you to the pagination controls, the you tab through those since the are buttons/selects.